src/pages/Contact.js

```diff
@@ -0,0 +1,14 @@
+import { GridContainer } from '@trussworks/react-uswds';
+import Form from '../components/ContactForm';
+
+function TeamPage() {
+  return (
+    <GridContainer containerSize="widescreen">
+      <br />
+      <h1>Contact Us</h1>
+      <Form />
+    </GridContainer>
+  );
+}
+
+export default TeamPage;
```

Comment:
Same as above. Did you maybe mean to export the ContactPage rather than the TeamPage in this file?